assets/schema_input.json

```diff
@@ -84,6 +84,6 @@
                 "pattern": "^[^\\n\\t\"]+$"
             }
         },
-        "required": ["sample", "mlst_alleles", "metadata_partition"]
+        "required": ["sample", "mlst_alleles"]
```

Comment:
For clarification, based on my understanding [here](https://github.com/phac-nml/clustersplitter/pull/9/commits/d2425b024a39e1549f6121489427b694511c43e7#r1704234324) I would assume the column is required. If it was empty would not all samples be excluded from being run? Or is this just to keep the pipeline halting if an entry is empty?

Reply:
The `required:` entry here is saying that each row in the sample sheet (each sample), must have entries in the `sample`, `mlst_alleles`, and (previously) `metadata_partition` columns. They cannot be empty (`""`).

The column must still exist in the sample sheet that's passed to the workflow, much like the other columns specified in the `schema_input.json` file, but it's (now) permissible to have any or all of the values in the `metadata_partition` column empty.

This behaviour is desired because (from what I understand) users often want to do select all their data, run it, and see the outputs, without having to bother manually removing individual samples that may be missing this field sometimes.

Reply:
Thanks for the clarification! I realize now it was more for my understanding than the PR :clown_face:. Thanks for taking the time to write it out.